src/index.js

```diff
@@ -1,12 +1,59 @@
 import "./style.css";
+import { createElement } from "./utils/elements";
+
+var startInterval;
+let counter = 400;
+
+function startTimer() {
+  if (!startInterval) {
+    startInterval = setInterval(countDownOne, 100);
+  }
+}
+
+function stopTimer() {
+  clearInterval(startInterval);
+}
+
+function countDownOne() {
+  counter--;
+  const counterNumber = document.querySelector(".timeOutput");
+  counterNumber.innerHTML = `${counter}ms`;
+  console.log(counterNumber.innerHTML);
+}
 
 function component() {
-  const element = document.createElement("div");
+  const timeOutput = createElement("span", {
+    className: "timeOutput",
+    innerHTML: `${counter}ms`,
+  });
+
+  const timeInput = createElement("input", {
+    className: "input--time",
+    placeholder: "hh:mm:ss",
+  });
+
+  const timerButtonStart = createElement("button", {
+    className: "button--startCount",
+    innerText: "Count Down",
+    onclick: () => {
+      startTimer();
+    },
```

Comment:
`startTimer` is already a function, thus, you don't need to create a new one.
```suggestion
    onclick: startTimer,
```